src/main/java/com/rocketden/main/dto/game/TesterResponse.java

```diff
@@ -3,10 +3,17 @@
 import lombok.Getter;
 import lombok.Setter;
 
+import java.util.List;
+
+import com.rocketden.main.game_object.SubmissionResult;
+
 @Getter
 @Setter
-// This class is mapped directly to the tester service's RunDto class.
 public class TesterResponse {
-    private Boolean status;
-    private String output;
+    // This class is mapped directly to the tester service's RunDto class.
+    private List<SubmissionResult> results;
```

Comment:
nit: Keep this as a separate class rather than reusing SubmissionResult, that way we can decouple the two services (so if tester changes its response object, we aren't forced to perfectly mirror the response in the main repo database

Reply:
Sounds good, will do!

Reply:
Done - there were a decent amount of Mapper changes and an additional test for the `TesterResult` class, so you may want to take a look at the new changes just to make sure everything looks ok.